app/cinema/hall.py

```diff
@@ -0,0 +1,19 @@
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
```

Comment:
```suggestion
            customers: list[Customer],
```

use more descriptive type annotations in all similar cases too